### PR TITLE
User view

### PR DESCRIPTION
**Pull request by milkovskamarija**
image
image

### CODE REVIEW

**Review by MuratUseini123 — APPROVED** [review]

File: src/app/app.module.ts [hunk 1]
```diff
@@ -63,8 +65,10 @@ import { AppointmentsListComponent } from './appointments-list/appointments-list
     RegisterComponent,
     ChangePasswordComponent,
     HomePageComponent,
-    LoginComponent,
-    AppointmentsListComponent
+    // LoginComponent,
```
Comment (StankovskiA) [inline src/app/app.module.ts:68]: Remove comments if they are not needed

File: src/app/app.module.ts [hunk 2]
```diff
@@ -63,8 +65,10 @@ import { AppointmentsListComponent } from './appointments-list/appointments-list
...
+    AppointmentsListComponent,
+    // LoginComponent,
```
Comment (StankovskiA) [inline src/app/app.module.ts:70]: Remove comments if they are not needed

File: src/app/core/services/login.service.ts [hunk 3]
```diff
@@ -9,7 +9,12 @@ import { environment } from 'src/environments/environment';
 export class LoginService {
 
   constructor(private http: HttpClient) { }
+
   loginUser(user: any): Observable<any> {
+    console.log("USER:");
```
Comment (StankovskiA) [inline src/app/core/services/login.service.ts:14]: No need of console logs in production

File: src/app/core/services/login.service.ts [hunk 4]
```diff
@@ -9,7 +9,12 @@ import { environment } from 'src/environments/environment';
...
+    
+    console.log(user);
```
Comment (StankovskiA) [inline src/app/core/services/login.service.ts:16]: No need of console logs in production

File: src/app/core/services/register.service.ts [hunk 5]
```diff
@@ -4,14 +4,20 @@ import { Observable } from 'rxjs';
...
 @Injectable({
-  providedIn: 'root'
+  providedIn: 'root',
 })
 export class RegisterService {
-
-  constructor(private http: HttpClient) { }
+  constructor(private http: HttpClient) {}
   createUser(user: any): Observable<any> {
+    console.log('Register ');
```
Comment (StankovskiA) [inline src/app/core/services/register.service.ts:12]: No need of console logs in production

**Review by StankovskiA — APPROVED** [review]

**Pull request merged**